### ISSUE TITLE
Special characters cause parser segmentation fault

### ISSUE THREAD

**Issue by hannahchiu** [emfomy/ckipnlp#18]
`Segmentation fault (core dumped)` occurs while `CkipParser(do_ws=False)` is called because inputs of parser contain halfwidth symbols ex.-

To Reproduce:
```
import ckipnlp.parser
ps = ckipnlp.parser.CkipParser(do_ws=False)
ps("103-105(Nd)")
```

**Comment by emfomy** [comment]
Since the C code of CKIPParser uses some half-width symbols as reserved words (keywords), the input sentences should not contain these characters.

Here is the list of known keywords: `( ) + - : : |`. One may replace those characters by full-width ones before calling CKIPParser.

Such utilities will be provided in later version.

**Comment by emfomy** [comment]
Implemented in 0.7.2.
